### ISSUE TITLE
Silently fail when aborting edition

### ISSUE THREAD

**Issue by LeBenLeBen** [sephii/taxi#65]
Since I use an external text editor, I can `CTRL + C` the edit command to keep the file open for the rest of the day.

It would be awesome if I don’t get a bunch of errors when I do this.

```
^CTraceback (most recent call last):
  File "/usr/local/bin/taxi", line 4, in <module>
    app.main()
  File "/usr/local/lib/python2.7/site-packages/taxi/app.py", line 211, in main
    app.run()
  File "/usr/local/lib/python2.7/site-packages/taxi/app.py", line 70, in run
    self.run_command(args[0], options.__dict__, command_args)
  File "/usr/local/lib/python2.7/site-packages/taxi/app.py", line 181, in run_command
    action.run()
  File "/usr/local/lib/python2.7/site-packages/taxi/commands.py", line 475, in run
    file.spawn_editor(self.options['file'], editor)
  File "/usr/local/lib/python2.7/site-packages/taxi/utils/file.py", line 35, in spawn_editor
    subprocess.call(editor)
  File "/usr/local/Cellar/python/2.7.8/Frameworks/Python.framework/Versions/2.7/lib/python2.7/subprocess.py", line 522, in call
    return Popen(*popenargs, **kwargs).wait()
  File "/usr/local/Cellar/python/2.7.8/Frameworks/Python.framework/Versions/2.7/lib/python2.7/subprocess.py", line 1376, in wait
    pid, sts = _eintr_retry_call(os.waitpid, self.pid, 0)
  File "/usr/local/Cellar/python/2.7.8/Frameworks/Python.framework/Versions/2.7/lib/python2.7/subprocess.py", line 476, in _eintr_retry_call
    return func(*args)
KeyboardInterrupt
```

**Comment by sephii** [comment]
Should be fixed in 4.0. Feel free to reopen if that's not the case.

**Comment by LeBenLeBen** [comment]
Wow much love very grateful
